api/src/pcapi/routes/backoffice/operations/blueprint.py

```diff
@@ -119,6 +119,15 @@ def create_event() -> utils.BackofficeResponse:
 @operations_blueprint.route("/<int:special_event_id>", methods=["GET"])
 @atomic()
 def get_event_details(special_event_id: int) -> utils.BackofficeResponse:
+    response_form = operations_forms.OperationResponseForm(formdata=utils.get_query_params())
+    if not response_form.validate():
+        flash(utils.build_form_error_msg(response_form), "warning")
+        return redirect(url_for("backoffice_web.operations.get_event_details", special_event_id=special_event_id), 303)
+
+    response_status_filter = []
+    if response_status_data := response_form.response_status.data:
+        response_status_filter = [operations_models.SpecialEventResponse.status.in_(response_status_data)]
```

Comment:
Détail : je verrais pour être plus clair : 
```suggestion
    response_status_filter = [operations_models.SpecialEventResponse.eventId == special_event_id]
    if response_status_data := response_form.response_status.data:
        response_status_filter.append(operations_models.SpecialEventResponse.status.in_(response_status_data))
```
et ne garder qu'un seul `filter` plus bas.